src/js/worklets/chat/messenger/messages.js

```diff
@@ -1,5 +1,5 @@
 import { useAnimatedReaction, withTiming, runOnJS } from 'react-native-reanimated';
-import { useState } from "react"
```

Comment:
these changes are needed? seems to just be a clash of formatter

Reply:
it was done by lint-fix

Reply:
my bad, forgot i didn't have a formatter for js on save :(. It's strange the CI didn't fail bc of this.